### PR TITLE
Clang refactoring

### PR DESCRIPTION
**Pull request by Vitozz**
Code modernize and refactoring using clan-analyzer from QtCreator and Clion

### CODE REVIEW

File: src/filecache.cpp [hunk 1]
```diff
@@ -217,8 +217,8 @@ FileCacheItem *FileCache::append(const QList<XMPP::Hash> &sums, const QByteArray
 {
     Q_ASSERT(sums.size() > 0);
 
-    FileCacheItem *item
-        = new FileCacheItem(this, sums, metadata, QDateTime::currentDateTime(), maxAge, size_t(data.size()), data);
+    FileCacheItem *item = new FileCacheItem(this, sums, metadata, QDateTime::currentDateTime(), maxAge,
+                                            qint64(size_t(data.size())), data);
```
Comment (Ri0n) [inline src/filecache.cpp:221]: better remove size_t cast at all

File: src/psicontactlistview.cpp [hunk 2]
```diff
@@ -55,10 +55,7 @@ class PsiContactListView::Private : public QObject {
         if (!allowAutoresize)
             return false;
 
-        if (lv->window()->isMaximized())
-            return false;
-
-        return true;
+        return !lv->window()->isMaximized() && true;
```
Comment (Ri0n) [inline src/psicontactlistview.cpp:58]: && true?

File: src/filecache.cpp (hunk 1 above)
Comment (Vitozz) [reply src/filecache.cpp]: Fixed

File: src/psicontactlistview.cpp (hunk 2 above)
Comment (Vitozz) [reply src/psicontactlistview.cpp]: Fixed

**Pull request merged**